ror-with-kibana-reverse-proxy-demo/init-scripts/utils/lib.sh

```
@@ -0,0 +1,50 @@
+  INDEX_NAME=$1
+  DOCUMENT_CONTENT=$2
+
+set -x
+
+  respone=$(curl -k -s -L -w "\n%{http_code}" -u $ELASTICSEARCH_USER:$ELASTICSEARCH_PASSWORD \
+    -X POST "$ELASTICSEARCH_ADDRESS/$INDEX_NAME/_doc/" \
+    -H "Content-Type: application/json" -d "$DOCUMENT_CONTENT"
+  )
+
+  http_status=$(echo "$respone" | tail -n 1)
+  response_body=$(echo "$respone" | sed \$d)
```

Shellcheck (0.11.0)

[warning] response_body appears unused. Verify use (or export if used externally).

(SC2034)